README.md

```diff
@@ -152,44 +164,36 @@ To add just the mogwais you want, use the `mogwai()` and `allMogwais()` method t
 
 ### Configuring Gremlins
 
-All the gremlins and mogwais provided by `gremlins.js` are _configurable functions_, i.e. you can alter the way they work by calling methods on them.
+All the gremlins and mogwais provided by `gremlins.js` are _configurable_, i.e. you can alter the way they work by injecting a custom configuration.
 
-For instance, the clicker gremlin is a function that you can execute it directly:
+For instance, the clicker gremlin is a function that take an object as custom configuration:
 
 ```js
-const clickerGremlin = gremlins.species.clicker();
-clickerGremlin(); // trigger a random mouse event in the screen:
-```
-
-In JavaScript, functions are objects, and as such can have methods. The clicker gremlin function offers customizing methods:
-
-```js
-gremlins.species
-    .clicker()
-    .clickTypes(['click']) // which mouse event types will be triggered
-    .canClick(function(element) {
-        // only click elements in bar
-        return $(element).parents('#bar').length;
-        // when canClick returns false, the gremlin will look for another
-        // element to click on until maxNbTries is reached
-    })
-    .showAction(function(x, y) {
-        // by default, the clicker gremlin shows its action by a red circle
-        // overriding showAction() with an empty function makes the gremlin action invisible
-    });
+const customClicker = gremlins.species.clicker({
+    // which mouse event types will be triggered
+    clickTypes: ['click'],
+    // Click only if parent is has class test-class
+    canClick: element => element.parentElement.className === 'test-class',
+    // by default, the clicker gremlin shows its action by a red circle
+    // overriding showAction() with an empty function makes the gremlin action invisible
+    showAction: (x, y) => {},
```

Comment:
```suggestion
    showAction: false
```

Reply:
It would work but I find it confusing. With this function, we understand that `showAction` take a function with 2 parameters: x, y. 
If we change it for false, it's unclear if `showAction` take a function or a boolean.